.github/workflows/main_template.yml

```diff
@@ -0,0 +1,37 @@
+on:
+    workflow_call:
+        inputs:
+            modulepath:
+              required: true
+              type: string
+        secrets:
+          codecov-token:
+            required: true
+
+# TODO: environment variables cannot be sent to reusable workflows: https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations
+# env:
+#   GO_VERSION: "1.22.x"
```

Comment:
remove?

maybe the only thing that can be done, is setting the Go version in a variable and then using substitution to pass it as the `go-version` variable.

Reply:
I would like to leave the comment until we apply a final solution or they allow to use env variables with workflows.